src/mo_values/prim.ml

```diff
@@ -241,6 +241,8 @@ let prim trap =
     end
   | "text_len" -> fun _ v k ->
     k (Int (Nat.of_int (List.length (Lib.Utf8.decode (Value.as_text v)))))
+  | "text_lowercase" -> fun _ v k ->
+    k (Text (String.lowercase_ascii (Value.as_text v))) (* TODO -- use Unicode here. *)
```

Comment:
this todo probably explains the test failing in interpreter (run, run-ir, run-low)